src/components/platformContent.tsx

```diff
@@ -32,9 +50,12 @@ export async function PlatformContent({includePath, platform, noGuides}: Props)
   }
 
   let doc: Awaited<ReturnType<typeof getFileBySlug>> | null = null;
+
   if (guide) {
+    let guidePath = `platform-includes/${includePath}/${guide}`;
+    guidePath = udpatePathIfVersionedFileDoesNotExist(guidePath);
```

Comment:
L: just a mini code style suggestion (same for the other occurrences in the file). I would rather refrain from using `let` if it is not strictly needed. 

```suggestion
    const guidePath = udpatePathIfVersionedFileDoesNotExist(`platform-includes/${includePath}/${guide}`);
```

Reply:
yeah good call